app/templates/components/forms/events/view/create-access-code.hbs

```diff
@@ -0,0 +1,66 @@
+<form class="ui form" {{action 'submit' on='submit'}}>
+  <div class="field">
+    <label class="required">{{t 'Access Code'}}</label>
+    {{input type='text' name='access_code' key-press='onChangeAccessCode'}}
```

Comment:
Follow to the comments I have added ... Bind this input to an a variable. And have a computed property maybe called as `accessUrl` that depends on this and let the URL change as the accessCode variable change.